types/src/network_address/mod.rs

```diff
@@ -531,10 +620,10 @@ impl Protocol {
             "dns6" => Protocol::Dns6(parse_one(args)?),
             "tcp" => Protocol::Tcp(parse_one(args)?),
             "memory" => Protocol::Memory(parse_one(args)?),
-            "ln-noise-ik" => Protocol::NoiseIK(x25519::PublicKey::from_encoded_string(
+            "noise-ik" => Protocol::NoiseIK(x25519::PublicKey::from_encoded_string(
                 args.next().ok_or(ParseError::UnexpectedEnd)?,
             )?),
-            "ln-handshake" => Protocol::Handshake(parse_one(args)?),
+            "handshake" => Protocol::Handshake(parse_one(args)?),
```

Comment:
We should note this is a breaking change, and old nodes won't work with new ones.